frontend/public/components/edit-yaml.jsx

```diff
@@ -299,7 +304,9 @@ export const EditYAML = connect(stateToProps)(
         action(model, obj, newNamespace, newName)
           .then(o => {
             if (redirect) {
-              history.push(this.props.redirectURL || resourceObjPath(o, referenceFor(o)));
+              const redirectURL =
+                _.isFunction(this.props.resourceObjPath) ? this.props.resourceObjPath(o) : this.props.redirectURL;
```

Comment:
This seems a bit weird, with `props.resourceObjPath` taking precedence over `props.redirectURL` which then takes precedence over `resourceObjPath()` call.

What I think is more correct - `props.redirectURL` should have the highest precedence.

```ts
let url = this.props.redirectURL;
if (!url) {
  const path = _.isFunction(this.props.resourceObjPath) ? this.props.resourceObjPath : resourceObjPath;
  url = path(o, referenceFor(o));
}
history.push(url);
```

Does that make sense?

Reply:
Nice ! tanks, fixed.